src/core/events.js

```diff
@@ -1,6 +1,7 @@
 import EventEmitter from 'eventemitter2'
 import store from '../store/store'
 import * as selectors from '../store/selectors'
+import { each } from 'lodash'
```

Comment:
More than a code review, maybe a topic for our next front end meeting, but if using ES6, I would strongly advice  against the use of libraries like Lodash, in favour of native methods. I think it will make our code better suited to endure the test of time.